{{ cookiecutter.repo_name }}/.cookiecutter/config

```diff
@@ -7,21 +7,21 @@ export ENVIRONMENT_TYPE={{ cookiecutter.venv_type }}
 
 VENV_PATH=.venv
 
-# uv configuration
 {% if cookiecutter.venv_type == 'uv' %}
+# uv configuration
 ENV_CREATE=uv venv ${VENV_PATH}
 ENV_ACTIVATE=source ${VENV_PATH}/bin/activate
 ENV_DEACTIVATE=deactivate
 ENV_REMOVE=rm -rf ${VENV_PATH}
-ENV_UPDATE=uv pip install .[dev]
+ENV_UPDATE=uv pip install ".[dev]"
```

Comment:
This can also simply be `uv sync` which will sync your environment to `pyproject.toml`. By default, `dev` dependencies are [included when syncing](https://docs.astral.sh/uv/concepts/projects/dependencies/#dependency-groups).

```suggestion
ENV_UPDATE=uv sync
```